src/ui-component/cards/Summary.js

```diff
@@ -1,22 +1,18 @@
 // material-ui
-import { Grid, Typography } from "@mui/material";
+import { Grid, Button, Typography } from "@mui/material";
 import SubCard from "./SubCard";
 
 // ==============================|| CUSTOM SUB CARD ||============================== //
 
 const Summary = () => {
     return (
-        <SubCard title="Summary">
-            <Grid container direction="column" spacing={1}>
-                <Grid item>
-                    <Typography variant="body2">
-                        Lorem ipsum dolor sit amen, consenter nipissing eli, sed do elusion tempos incident ut laborers et doolie magna
-                        alissa. Ut enif ad minim venice, quin nostrum exercitation illampu laborings nisi ut liquid ex ea commons construal.
-                        Duos aube grue dolor in reprehended in voltage veil esse colum doolie eu fujian bulla parian. Exceptive sin ocean
-                        cuspidate non president, sunk in culpa qui officiate descent molls anim id est labours.
-                    </Typography>
-                </Grid>
-            </Grid>
+        <SubCard title="<Group Name>">
```

Comment:
Do we want to make 1 reusable Summary component (and pass values with props) to use across both My Groups and Individual Group? Or do we want to make 2 separate summary component files for each?

Reply:
2 Separate for now since I dont have a My groups page yet